imgui_bezier_math.h

```diff
@@ -25,8 +26,21 @@ struct ImCubicBezierPointsT
     T P2;
     T P3;
 };
-using ImCubicBezierPoints = ImCubicBezierPointsT<ImVec2>;
 
+//------------------------------------------------------------------------------
+template <typename T>
+struct ImLinePointsT
```

Comment:
Suggestion for the future:
```
template <typename T>
using ImLinePointsT = std::array<T, 5>;
```
or even just
```
using ImLinePoints = std::array<ImVec2, 5>;
```
that would allow to simply loop over the points in the code below.
Since you have to deal with the bezier case as well, this might not be a good idea to do just now, but cleans up code in the long run.